content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -0,0 +1,217 @@
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Overview
+
+Session Replay provides privacy controls to ensure organizations of any scale do not expose sensitive or personal data. Data is stored on Datadog-managed cloud instances and encrypted at rest.
+
+Default privacy options for Session Replay are designed to protect end user privacy and prevent sensitive organizational information from being collected.
+
+By enabling Mobile Session Replay, you can automatically mask sensitive elements from being recorded through the RUM Mobile SDK. When data is masked, that data is not collected in its original form by Datadog's SDKs and thus is not sent to the backend.
+
+## Configuration
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+[ add android content]
+
+   {{< code-block lang="javascript" filename="build.gradle" disable_copy="false" collapsible="true" >}}
+
+   {{< /code-block >}}
+
+{{% /tab %}}
+
+{{% tab "iOS" %}}
+
+   {{< code-block lang="swift" filename="AppDelegate.swift" disable_copy="false" collapsible="true" >}}
+
+   // mask all text elements
+   let configuration = SessionReplayConfiguration(privacy: .maskAll)
+   sessionReplay = SessionReplay.initialize(with: configuration)
+   sessionReplay.start()
+
+   // mask only input elements
+   let configuration = SessionReplayConfiguration(privacy: .maskUserInput)
+   sessionReplay = SessionReplay.initialize(with: configuration)
+   sessionReplay.start()
+
+   // no masking; all text is revealed
+   let configuration = SessionReplayConfiguration(privacy: .allowAll)
+   sessionReplay = SessionReplay.initialize(with: configuration)
+   sessionReplay.start()
+
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Masking modes
+
+By default, the `mask all` setting is enabled for all data. Using the masking modes below, you can override the default setup on a per-application basis.
+
+| Masking mode | Description | Example |
+|--------------|-------------|---------|
+| `mask all` | All text content on screen is replaced with X's. | {{< img src="real_user_monitoring/session_replay/mobile/masking-mode-mask-all.png" alt="Enable mask all to replace all text content on the screen with X's" style="width:40%;">}} |
+| `mask user input` | Any input field is replaced with anonymized text. | {{< img src="real_user_monitoring/session_replay/mobile/masking-mode-mask-user-input.png" alt="Enable mask all to replace all text content on the screen with X's" style="width:40%;">}} |
+| `allow all` | All text is revealed. <br> <br> **Note**: Even with this option enabled, any sensitive text fields such as passwords, emails, phone numbers, and addresses are still masked. | {{< img src="real_user_monitoring/session_replay/mobile/masking-mode-allow-all.png" alt="Enable mask all to replace all text content on the screen with X's" style="width:40%;">}} |
+
+### Unmask data in all recorded content
```

Comment:
@rtrieu @mariusc83 IMO this section is now repetitive: in the beginning we will talk about the code snippets for the different masking settings (like how iOS has) and thus that will cover how to unmask data in recorded content. thoughts on cutting this?

Reply:
Yeah, let me consolidate this content.